.github/workflows/convert_to_python_script.yml

```diff
@@ -11,6 +11,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
```

Comment:
Gui, estava aqui olhando a Action que faz isso e vi:

> By default, this Action will not run on Pull Requests which have been opened by forks. (This is a limitation by GitHub, not by us.)

https://github.com/stefanzweifel/git-auto-commit-action#using-the-action-in-forks-from-public-repositories

Dá uma lida e vamos discutir um pouco sobre isso.

Reply:
Outras alternativas além dessa Action:

* Criar um git hook: https://github.com/jupyterhub/jupyterhub/issues/1412
* Criar um teste unitário que verifica os nossos notebooks e falha se o script Python correspondente não for encontrado